libs/local_lcos/CMakeLists.txt

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+# Default location is $HPX_ROOT/libs/local_lcos/include
+set(local_lcos_headers
+  hpx/local_lcos.hpp
+  hpx/local_lcos/and_gate.hpp
+  hpx/local_lcos/barrier.hpp
+  hpx/local_lcos/channel.hpp
+  hpx/local_lcos/composable_guard.hpp
+  hpx/local_lcos/conditional_trigger.hpp
+  hpx/local_lcos/condition_variable.hpp
+  hpx/local_lcos/counting_semaphore.hpp
+  hpx/local_lcos/detail/condition_variable.hpp
+  hpx/local_lcos/detail/counting_semaphore.hpp
+  hpx/local_lcos/detail/sliding_semaphore.hpp
+  hpx/local_lcos/event.hpp
+  hpx/local_lcos/futures_factory.hpp
```

Comment:
Can or should `hpx/future.hpp` also be in this module?

Reply:
I'll do a `future` module, I removed `futures_factory.hpp` from this module, that was a mistake ^^